async_downloads/templates/async_downloads/ws_downloads.html

```diff
@@ -0,0 +1,37 @@
+{% load sekizai_tags static humanize i18n %}
```

Comment:
I think we shouldn't need a separate template for websockets version - they should both use the same template. A user of the package can always override the template if they want to, and if we just need to add an additional class (i.e. my `app_name` -> `css_class` comment), we can do that for both websocket and http styles.

Reply:
8a54d5fed6559a144d441b92e81fba65db7069f6